src/GraphQL/FromUserContextAttribute.cs

```diff
@@ -0,0 +1,18 @@
+using GraphQL.Execution;
+using GraphQL.Types;
+
+namespace GraphQL
```

Comment:
Does your IDE suggest you to use file scoped namespaces?

Reply:
No, at least, not as a warning.  But I can use VS to easily refactor as a file-scoped namespace.  (As it is now this file matches the rest of the codebase.)

Reply:
I configured this rule  as suggestion so VS should respect it when creating new files.

Reply:
Ok but let's keep the codebase uniform.  In this case, the file was copied from `FromSourceAttribute` and slightly modified.

Reply:
ok